src/App.test.js

```diff
@@ -1,8 +1,126 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import Calculate from './logic/calculate';
```

Comment:
- [ ] Again here it is a requirement that we test all the functionalities and scenarios of `calculate.js`, 

- For example test for the scenario where the user changes a number from positive to negative.
- Also test for the scenario where the user presses the `AC` button to reset the app.